guides/ansible_engine/4-runplaybook/README.md

```diff
@@ -0,0 +1,37 @@
+# Exercise 4 - Running the apache-basic-playbook
+
+Congratulations!  You just wrote a playbook that incorporates some key Ansible concepts that you use in most if not all of your future playbooks.  Before you get too excited though, we should probably make sure it actually runs.
+
+So, lets do that now.
+
+## Section 1 - Running your new apache playbook
+
+### Step 1:
+Make sure you are in the right directory and create a host file.
+
+```bash
+cd ~/apache-basic-playbook
+```
+
+---
+**NOTE**
+Since you already did the work of creating a host file in Lession 1.0, feel free to re-use it.
```

Comment:
Thinking this is really unnecessary. It's restating the obvious at best. As previously mentioned they shouldn't even be doing it because the provisioner handles generating it for them. Guess this is a comment and not a requested change yet though.

Reply:
Agreed. We can clean up all the commentary / comments to keep it to the point and precise.  There is humor sprinkled throughout the guide but with this being available to everyone to use to educate themselves and others, we should pull it back to being strictly professional